dev/AppLifecycle/ActivationRegistrationManager.cpp

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#include <pch.h>
+#include <ActivationRegistrationManager.h>
+#include <ActivationRegistrationManager.g.cpp>
+
+#include "LaunchActivatedEventArgs.h"
+#include "ProtocolActivatedEventArgs.h"
+#include "Shared.h"
+
+namespace winrt::Microsoft::ProjectReunion::implementation
+{
+    void ActivationRegistrationManager::RegisterForFileTypeActivation(hstring const& groupName,
+        hstring const& logo, array_view<hstring const> supportedFileTypes,
+        array_view<hstring const> supportedVerbs)
+    {
+        throw hresult_not_implemented();
+    }
+
+    void ActivationRegistrationManager::RegisterForProtocolActivation(hstring const& scheme,
+        hstring const& displayName)
+    {
+        if (scheme.empty() || displayName.empty())
+        {
+            throw winrt::hresult_invalid_argument();
+        }
+
+        if (HasIdentity())
```

Comment:
Our only scenario for release 0.1 is packaged win32, which has identity.  Will this cause it to not work?  Or protocol activation is not supported currently for packaged apps?

Reply:
Dynamic registration (@ runtime) of extension contracts will not be supported for packaged applications.  Instead they require entries to be placed in their manifest in order to register.